Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner(props) {
+    return (
+        <div class='card-banner'>
```

Comment:
Open up your console in the chrome tools when you're viewing this. It will give you an error message that says something like `use className instead of class`. In react, you have to use `className` instead of `class` because class is a reserved keywork in Javascript. `card-banner` will have to change to camelCase too, `cardBanner` because javascript reads that as a minus sign if you don't change it.